app/views/components/widget/_fba2.js.erb

```diff
@@ -82,6 +85,21 @@ function FBAform(d, N) {
 				d.head.appendChild(style);
 			}
 		},
+		_loadUswds: function() {
```

Comment:
I'm running up against my Rail inexperience here and I'm open to other ways of loading these 2 static files for the widget, including ways that maintain the single downloadable javascript payload. Requesting these assets separately seems reasonable to me - I don't think it will have a noticeable impact on performance, assuming we minify and cache the assets, maybe even put them in a CDN one day. The only users who might notice are those with `load_css=true` who are self-hosting a copy of their widget's javascript.